extras/nest_vars.sh.in

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh
 
-# NEST is installed here. When you relocate NEST, change this variable.
-export NEST_INSTALL_DIR="@CMAKE_INSTALL_PREFIX@"
+# NEST is installed here. When you relocate NEST, change these variables.
+NEST_LIB_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/nest/"
+NEST_BIN_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_FULL_BINDIR/"
```

Comment:
Doesn't *_FULL contain the absolute path?

Reply:
You're right. Thanks for noticing. The following change should fix this:
```suggestion
NEST_BIN_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR/"
```